src/content/app/genome-browser/state/api/queries/geneSummaryQuery.ts

```diff
@@ -83,9 +90,9 @@ type GeneSummary = Pick<
 > & {
   metadata: GeneMetadata;
 } & Pick3<FullGene, 'slice', 'strand', 'code'> &
-  Pick3<FullGene, 'slice', 'location', 'length'> & {
+  Pick3<FullGene, 'slice', 'location', 'length' | 'start' | 'end'> & {
     transcripts: Transcript[];
-  };
+  } & Pick4<FullGene, 'slice', 'region', 'sequence', 'checksum'>;
```

Comment:
let's move this line immediately after line 93

Reply:
Done